kong/datadog_checks/kong/metrics.py

```diff
@@ -18,4 +19,16 @@
     'kong_nginx_http_current_connections': 'nginx.http.current_connections',
     'kong_nginx_stream_current_connections': 'nginx.stream.current_connections',
     'kong_stream_status': 'stream.status',
+    # Kong 2.8
+    'kong_nginx_timers': 'nginx.timers',
```

Comment:
I don't see a reference to this in metadata.csv.

Reply:
This one looks like it was added in v2.8 ([source](/Users/steven.wenzel/DatadogRepos/integrations-core/kong/tests/test_unit.py)), updated the metadata file accordingly.